roles/acc/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+# defaults file for acc
+acc_context: control-center
+acc_port: 8881
+app_config_json_path: "{{ nginx_default_vhost_docroot }}/{{ acc_context }}/app.config.json"
+app_default_configuration:
+  APP_CONFIG_PROVIDER: ECM
+  APP_CONFIG_ECM_HOST: "{protocol}//{hostname}{:port}"
+  APP_CONFIG_BPM_HOST: "{protocol}//{hostname}{:port}"
+  APP_CONFIG_AUTH_TYPE: BASIC
+  APP_CONFIG_IDENTITY_HOST: "{protocol}//{hostname}{:port}/auth/realms/alfresco"
+  APP_CONFIG_OAUTH2_HOST: "{protocol}//{hostname}{:port}/auth/realms/alfresco"
+  APP_CONFIG_OAUTH2_CLIENTID: alfresco
+  APP_CONFIG_OAUTH2_IMPLICIT_FLOW: true
+  APP_CONFIG_OAUTH2_SILENT_LOGIN: true
+  APP_CONFIG_OAUTH2_REDIRECT_SILENT_IFRAME_URI: "{protocol}//{hostname}{:port}/assets/silent-refresh.html"
+  APP_CONFIG_OAUTH2_REDIRECT_LOGIN: "/"
+  APP_CONFIG_OAUTH2_REDIRECT_LOGOUT: "/"
```

Comment:
I must replace all the properties to avoid a broken json, optionally strings can be set to empty, if they don't break the application itself.

anyway those values you see are the same that I can find in adw app config - so at least they would be consistent